src/components/previews/PreviewPlayer.vue

```diff
@@ -1589,9 +1554,11 @@ export default {
       if (this.currentPreview.annotations) {
         const annotations = []
         if (this.currentPreview.annotations.forEach) {
-          this.currentPreview.annotations.forEach(a =>
-            annotations.push({ ...a })
-          )
+          this.currentPreview.annotations.forEach(a => {
+            if (a.time >= 0) {
+              annotations.push({ ...a })
+            }
+          })
```

Comment:
You can use an `Array.filter()` here:

```js
const annotations = this.currentPreview.annotations.filter(a => a.time >= 0);
```

Reply:
It's not possible because we need to clone the annotation.

Reply:
But `filter()` create a shallow copy similar to the spread operator `{... a}`